### ISSUE TITLE
Password change request : Invalid Challenge

### ISSUE THREAD

**Issue by steveyp1987** [jpatokal/openflights#820]
Hi, 
When I click on the link to change my password, it simply says "invalid Challenge". I can see someone had this issue back in 2016 but the case was closed. Please help - thank you! 

![image](https://user-images.githubusercontent.com/34452319/33840502-04d89a74-de8d-11e7-87cf-9d145ffaae9d.png)

**Comment by jpatokal** [comment]
New airport edit suggestion submitted by bandrzej:

UPDATE airports SET name='Amanwana seaplane base', city='Moyo Island', country='Indonesia', iata=NULL, icao=NULL, x=117.497031, y=-8.2633644, elevation=0, timezone=8, dst='N' WHERE apid=2984;

Existing, potentially conflicting airport information:

```
Array
(
    [name] => Bryansk Airport
    [city] => Bryansk
    [country] => Russia
    [iata] => BZK
    [icao] => UUBP
    [x] => 34.176399231
    [y] => 53.214199066199996
    [elevation] => 663
    [apid] => 2984
    [uid] => 
    [timezone] => 3
    [dst] => N
    [tz_id] => Europe/Moscow
    [type] => airport
    [source] => OurAirports
)

```

Cross-check this edit on other sites with compatible licensing:
- OurAirports: http://ourairports.com/airports//pilot-info.html
- Wikipedia: http://www.google.com/search?q=wikipedia%20%20airport&btnI

**Comment by jpatokal** [comment]
New airport edit suggestion submitted by bandrzej:

UPDATE airports SET name='Amanwana seaplane base', city='Moyo Island', country='Indonesia', iata=NULL, icao=NULL, x=117.497031, y=-8.2633644, elevation=0, timezone=8, dst='N' WHERE apid=2984;

Existing, potentially conflicting airport information:

```
Array
(
    [name] => Bryansk Airport
    [city] => Bryansk
    [country] => Russia
    [iata] => BZK
    [icao] => UUBP
    [x] => 34.176399231
    [y] => 53.214199066199996
    [elevation] => 663
    [apid] => 2984
    [uid] => 
    [timezone] => 3
    [dst] => N
    [tz_id] => Europe/Moscow
    [type] => airport
    [source] => OurAirports
)

```

Cross-check this edit on other sites with compatible licensing:
- OurAirports: http://ourairports.com/airports//pilot-info.html
- Wikipedia: http://www.google.com/search?q=wikipedia%20%20airport&btnI

**Comment by chrisrosset** [comment]
Let's use #1062 for tracking this - it's the more recent occurrence.

**Comment by dougj365** [comment]
What was the resolution to this issue?  I'm having the same problem but can't find an answer here or at #1062

**Comment by reedy** [comment]
> What was the resolution to this issue? I'm having the same problem but can't find an answer here or at #1062

It was closed in favour of #1062.

#1062 isn't resolved, so there's no resolution.
